README.md

```diff
@@ -1,179 +1,47 @@
-# 🚀 api
```

Comment:
What if you symlink'd to the subpackage's README.md? I just looked in Next's monorepo and discovered they did this: https://github.com/vercel/next.js/blob/canary/readme.md

Reply:
great idea